data/langdb.yaml

```diff
@@ -291,7 +291,7 @@ languages:
   li: [Latn, [EU], Limburgs]
   lij: [Latn, [EU], Ligure]
   liv: [Latn, [EU], Līvõ kēļ]
-  lki: [Arab, [AS, ME], لەکی‎]
+  lki: [Arab, [AS, ME], لەکی]
```

Comment:
This line removes an invisible character 200E, [left to right mark](https://en.wikipedia.org/wiki/Left-to-right_mark). It's never supposed to be needed in names of languages. (It _is_ used in MediaWiki's own Names.php, but for a very bad reason, which is irrelevant here.)